src/Data/List/Relation/Binary/Disjoint/Propositional/Properties.agda

```diff
@@ -8,13 +8,14 @@
 
 module Data.List.Relation.Binary.Disjoint.Propositional.Properties where
 
+open import Agda.Builtin.List using (List; []; _∷_)
```

Comment:
We shouldn't **add** imports to `Agda.Builtin.List` (or any other `Agda.Builtin`. These should only be imported in `Data.X.Base`, which then re-exports them. So `using (List; []; _∷_)` should be moved to `Data.List.Base` below.

Reply:
Yes, this looks like legacy code.
If the module imports *both* `Agda.Builtin.List` *and* `Data.List.Base` then something is wrong. 
if the `Builtin` import can be deleted safely, please do so!

Reply:
I think this was marked as resolved prematurely

Reply:
My mistake, my browser had been showing me cached version of these pages... sigh.